x-pack/plugins/stack_connectors/public/connector_types/gen_ai/logo.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { LogoProps } from '../types';
+
+const Logo = (props: LogoProps) => (
```

Comment:
Could you please do a `common` export of this logo as well? We're using this in the connector setup UI (since that's custom and not re-using the existing connector flyout/accordion).

Reply:
Done, you can now import `GenAiLogo` from `stack_connectors/public/common`